### DISCUSSION TITLE
problem with dot

### DISCUSSION THREAD

**Discussion by danielchmuro** [PrateekKumarSingh/AzViz#113] [Q&A]
i have this error message
Get-Command : The term 'dot' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the na
me, or if a path was included, verify that the path is correct and try again.
At C:\Users\danko\AzViz\AzViz\src\private\Get-DOTExecutable.ps1:9 char:32
+ ... ibleGraphVizPaths += (Get-Command -Type Application -Name dot).Source
+                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (dot:String) [Get-Command], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand

can anyone help?

**Comment by danielchmuro** [comment]
Problem solved. You can close the issue.

**Reply by BenoitBEAUCHAMP** [reply]
Hello, How ?
